src/main/webapp/app/pool/pool-booking/pool-booking.component.ts

```diff
@@ -10,16 +11,44 @@ import { PoolBookingService } from 'app/pool/pool-booking/pool-booking.service';
 export class PoolBookingComponent implements OnInit {
     public reservationPoolId: string;
     public count: number;
+    tools: Tool[];
+    public selectedTools: string[];
 
-    constructor(private route: ActivatedRoute, private poolBookingService: PoolBookingService) {}
+    constructor(private route: ActivatedRoute, private poolBookingService: PoolBookingService, private elementRef: ElementRef) {}
 
     ngOnInit(): void {
         this.route.paramMap.subscribe(e => (this.reservationPoolId = e.get('id')));
         this.count = 1;
+        this.poolBookingService.getTools(this.reservationPoolId).subscribe(tools => {
+            this.tools = tools.body;
+            this.createToolList();
+        });
+        this.selectedTools = [];
+    }
+
+    private createToolList(): void {
+        const select = this.elementRef.nativeElement.querySelector('#toolsSelected');
+        for (const tool of this.tools) {
+            const option = '<option>' + tool.name + '-' + tool.version + '</option>';
+            select.insertAdjacentHTML('beforeend', option);
+        }
     }
 
     book(): void {
         this.poolBookingService.book(this.reservationPoolId, this.count).subscribe(this.previousState);
+        this.sendToolsRequest();
+    }
+
+    private sendToolsRequest(): void {
+        const select = this.elementRef.nativeElement.querySelector('#toolsSelected');
+        for (const option of select) {
```

Comment:
Nah, do it like this:
[https://ng-select.github.io/ng-select#/multiselect](https://ng-select.github.io/ng-select#/multiselect)

Your solution is hacky and it can be done in a simple way